drivers/dac/ad3530r/ad3530r.c

```diff
@@ -0,0 +1,1093 @@
+/**************************************************************************//**
```

Comment:
Use the BSD 3 clause license instead (e.g https://github.com/analogdevicesinc/no-OS/blob/main/include/no_os_aout.h#L1).